Worker/Worker.java

```diff
@@ -85,8 +85,14 @@ public static String getTweet(String url) throws BadURLException{
 			doc = Jsoup.connect(url).get();
 		} catch (IOException e) {
 			throw new BadURLException(e);
-		}
-		return doc.title().split("\"")[1];
+        }
+
+        if (!doc.title().contains("on Twitter:"))
+        {
+            throw new BadURLException(new Exception("Bad Tweet!" + doc.title()));
+        }
+
+        return doc.title().split("\"")[1];
```

Comment:
some of the tweets they gave us are private; this cause the 'split' to return an empty array, and we crash. the above fixed it.